client/payment-details/dispute-details/style.scss

```diff
@@ -48,3 +48,41 @@
 		margin-bottom: 8px;
 	}
 }
+.dispute-steps {
+	&__header {
+		font-weight: 600;
+	}
+	&__steps {
+		list-style-position: inside;
+		margin-left: 0;
+
+		.wcpay-tooltip__content-wrapper.wcpay-tooltip--click__content-wrapper
+			> .wcpay-tooltip__content-wrapper
+			> [role='button'] {
+			margin-left: 0;
+			margin-right: 0;
+			vertical-align: text-bottom;
```

Comment:
This is so the icon is positioned just right.

https://github.com/Automattic/woocommerce-payments/assets/73803630/497651c6-c437-4936-8383-8f1621d76aa2

Reply:
I've also moved the `vertical-align: text-bottom` as a default tooltip style in 6a55603